src/components/ReplyForm/ReplyForm.tsx

```diff
@@ -22,7 +23,20 @@ export type Props = ReplyFormProps &
     ContainerProps &
     Pick<FormikProps<FormValues>, 'errors' | 'setFieldValue' | 'values'>;
 
+// export for test purpose
+export const handleKeyDown = (handleCancel: () => void) => (event: KeyboardEvent): void => {
```

Comment:
We should avoid testing the internals of the component whenever possible. Can we mount this component and set up a keydown listener on its parent document to make sure it doesn't get hit?